### PR TITLE
Word Boggle (Backtracking Problem) Solution

### PR DESCRIPTION
**Pull request by utopian-coder**
## Proposed changes

Added solution for a popular backtracking problem, called - Word Boggle.

## Types of changes

What types of changes does your code introduce to this project?
_Put an `x` in the boxes that apply_

- [ ] Proofreading (fix spelling, grammar, punctuation, accuracy) [More about Proofreading](https://www.scribbr.com/language-rules/what-is-proofreading/)
- [`x` ] New Files/Folder (adding folder without removing existing one)
- [ ] Add/Remove Files (adding/removing existing files/folder)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. Don't hesitate to ask if you're unsure about any of them. We're here to help! This is simply a reminder of what we will look for before merging your code._

- [x] Tested any code against its gcc/clang or the official binaries and it passes locally with my changes
- [ `x`] I have added the necessary explanation
- [ ] I have added the necessary links in the working section
- [`x` ] Added external links

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### Reviewers: 

This will be filled by the people who review this particular Pull Request.

### CODE REVIEW

File: Algorithms/Word Boggle/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+U E K
+Q S E
+Words we got is denoted using same color.
+```
+
+## Solution
+
+```cpp
+//Back-end complete function Template for C++
+class Solution {
```
Comment (shhossain) [inline Algorithms/Word Boggle/README.md:43]: code showing error

**Review by shhossain — CHANGES_REQUESTED** [review]
Fix the issue.

File: Algorithms/Word Boggle/README.md (hunk 1 above)
Comment (shhossain) [inline Algorithms/Word Boggle/README.md:45]: Code not running.

**Comment by utopian-coder** [comment]
May be it's because I haven't given the driver code (Which is kind of irrelevant I thought). So should I update it?

**Comment by shhossain** [comment]
@utopian-coder yes. You should also add header files.

**Comment by utopian-coder** [comment]
Added the headers ✅

**Review by shhossain — APPROVED** [review]

**Review by shhossain — APPROVED** [review]

**Pull request merged**